tests/test_lint_recipe.py

```diff
@@ -817,6 +817,40 @@ def test_string_source(self):
         ).format(type(url).__module__, type(url).__name__)
         self.assertIn(msg, lints)
 
+    def test_single_space_pins(self):
+        meta = {
+            "requirements": {
+                "build": [
+                    "{{ compilers('c') }}",
+                    "python >=3",
+                    "pip   19",
+                ],
+                "host": [
+                    "python >= 2",
+                ],
+                "run": [
+                    "xonsh>1.0",
+                    "conda= 4.*",
+                    "conda-smithy<=54.*",
+                ],
+            }
+        }
+        lints, hints = linter.lintify(meta)
+        filtered_lints = [lint for lint in lints if lint.startswith("``requirements: ")]
+        expected_messages = [
+            "``requirements: build: pip   19`` should only contain a single space "
+            "between the name and the pin, i.e. ``pip 19``",
```

Comment:
This is not necessary and we would need to change many recipes for no reason.

Reply:
Consistency is valuable. We have other lints for spacing

Reply:
Then those should be hints. Other lints for spacing have been there since the beginning, so there'll be no existing recipes failing. What this PR does is failing good recipes for no reason. How about making this a hint?

Reply:
That seems fair.

Reply:
Lint or do not, there is no hint ;-p

Reply:
Having just had to [update a recipe](https://github.com/conda-forge/fastapi-feedstock/pull/5#issuecomment-528131801) for no good reason I'd be against this change - it seems pointless.

However this ship has sailed so ¯\\_(ツ)_/¯

Reply:
@ocefpaf, would you be able to make a new release with https://github.com/conda-forge/conda-smithy/pull/1137 ?

Reply:
Yep. I'll do it as soon as I switch back to Linux. (On Windows building qt now.)